test/allowlist_for_publicAPI.json

```diff
@@ -704,6 +704,10 @@
     "Any",
     "Callable"
   ],
+  "torch.distributed.algorithms.ddp_comm_hooks.powerSGD_hook": [
```

Comment:
Nothing should be added to this list.
You should make sure to only expose what is needed.

Since you have `__all__` defined here, I don't think these two are needed at all.

Reply:
Reverted. Yesterday tests kept failing until I added these lines. It looks good now.

Reply:
Yes a small fix landed for that yesterday. Sorry for the noise :/